docs/PROTOCOL.rst

```diff
@@ -924,17 +954,18 @@ Get a list of features and services supported by the server.
       "hash_function": "sha256"
   }
 
-Protocol Version 1.2
---------------------
-
 Protocol version 1.2 introduces new methods `blockchain.block.headers`,
 `mempool.get_fee_histogram`.
 
-`blockchain.block.get_chunk` and all methods beginning
- `blockchain.address.` are deprecated and support will be removed in
- some future protocol version.  You should update your code to use
- `blockchain.block.headers` and `Script Hashes`_ with the scripthash
- methods introduced in protocol 1.1 instead.
+* `blockchain.block.get_chunk` and all methods beginning `blockchain.address.` are deprecated
+  and support will be removed in some future protocol version.  You should update your code to use `blockchain.block.headers`
+  and `Script Hashes`_ with the scripthash methods introduced in protocol 1.1 instead.
+
+* `blockchain.transaction.get` now have the optional parameter *verbose*. In verbose mode (default=False) the
+  result is enriched with the relative inclusion blockhash, if any. Feature may have be unavailable.
```

Comment:
Please describe this as simply returning what the network daemon returns; we will not make any guarantees about what that might be.

Reply:
Ok :-)